src/sentry/tagstore/snuba/backend.py

```diff
@@ -330,7 +330,7 @@ def get_group_ids_for_users(self, project_ids, event_users, limit=100):
         result = snuba.query(start, end, ['issue'], conditions, filters,
                              aggregations, limit=limit, orderby='-seen',
                              referrer='tagstore.get_group_ids_for_users')
-        return result.keys()
+        return set(result.keys())
 
     def get_group_tag_values_for_users(self, event_users, limit=100):
```

Comment:
This method (`get_group_tag_values_for_users`) but as far as I can tell, it's never been called while we've been tracking results.